src/otx/core/model/entity/detection.py

```diff
@@ -338,6 +353,16 @@ def _customize_outputs(
         bboxes = []
         scores = []
         labels = []
+
+        # some OMZ model requires to shift labels
+        first_label = (
+            self.model.model.get_label_name(0)
+            if isinstance(self.model, DetectionTiler)
+            else self.model.get_label_name(0)
+        )
+
+        label_shift = 1 if first_label == "background" else 0
```

Comment:
Please warn to an user if the label shift happens.

Reply:
Warning added: https://github.com/openvinotoolkit/training_extensions/pull/2925/files#diff-21305702e6b48749df9867a5a7cd30ad21f2099e1055fb1d522e42b5fe03e6e0R382